lib/quest/quest_watcher.rb

```diff
@@ -35,43 +34,39 @@ def run_specs
       notifications = loader.send(:notifications_for, RSpec::Core::Formatters::JsonFormatter)
       reporter.register_listener(formatter, *notifications)
       # End workaround
-      
-      # Get the current quest spec
-      spec_file = File.join(quest_dir, active_quest, "#{active_quest}_spec.rb")
 
       # Run the test
-      Quest::LOGGER.info("Running tests in #{spec_file}")
+      Quest::LOGGER.info("Beginning run of tests in #{spec_file}")
       RSpec::Core::Runner.run([spec_file])
-      
+
       # Store test results
-      output_file = File.join(STATE_DIR, "#{active_quest}.json")
-      Quest::LOGGER.info("Writing RSpec output to #{output_file}")
       File.open(output_file, "w"){ |f| f.write(formatter.output_hash.to_json) }
```

Comment:
I think this will break because you're changing the variable name to "changed_file_path"